files/workflows/ZWEWRF04.xml

```diff
@@ -168,6 +189,8 @@
         <template>
             <inlineTemplate substitution="true">set -x
 cd "${instance-zowe_build}/install"
+TMPDIR="$!{instance-zowe_user_temp}"
+export TMPDIR="$TMPDIR"
```

Comment:
zowe-user-temp is set as optional, so provide a default here
TMPDIR="$!{instance-zowe_user_temp}"
export TMPDIR="${TMPDIR:-/tmp}"

Reply:
There is already default in `zowe-install.sh` on line 55:

```sh
export TEMP_DIR=${TMPDIR:-/tmp}/zowe_"`date +%Y-%m-%d`"
```

Putting a default on the `TMPDIR` would overwrite the aforementioned default.